src/test/java/constants/ByConstants.java

```diff
@@ -0,0 +1,18 @@
+package constants;
+
+import org.openqa.selenium.By;
+
+/**
+ The purpose of the ByConstants class is to list all possible types of locators in Selenium WebDrive
+ This is not applicable to the project
+ */
+public class ByConstants {
+    public static final By BY_CLASSNAME = By.className("information");
+    public static final By BY_CSS_SELECTOR = By.cssSelector("#fname");
+    public static final By BY_ID = By.id("id");
+    public static final By BY_NAME = By.name("newsletter");
+    public static final By BY_LINK_TEXT = By.linkText("Selenium Official Page");
+    public static final By BY_PARTIAL_LINK_TEXT = By.partialLinkText("Official Page");
+    public static final By BY_TAG_NAME = By.tagName("a");
+    public static final By BY_XPATH = By.xpath("//input[@value='f']");
```

Comment:
let's have names which corresponds to their purpose, like `NAME_SELECTOR`

Reply:
Done!

Reply:
If you meant that I need to name selectors meaningfully (for example, SubmitButton), than I need to create another locators because these one are totally random and non-applicable in application.